backend/db/seeds.rb

```diff
@@ -1,71 +1,307 @@
+          photo_url = partial_photo_url['1:1']['1600']
+        elsif partial_photo_url['1:1']['800']
+          photo_url = partial_photo_url['1:1']['800']
+        elsif partial_photo_url['1:1']['400']
+          photo_url = partial_photo_url['1:1']['400']
+        elsif partial_photo_url['1:1']['200']
+          photo_url = partial_photo_url['1:1']['200']
+        end
+      elsif partial_photo_url['4:3']
+        if partial_photo_url['4:3']['1600']
+          photo_url = partial_photo_url['4:3']['1600']
+        elsif partial_photo_url['4:3']['800']
+          photo_url = partial_photo_url['4:3']['800']
+        elsif partial_photo_url['4:3']['400']
+          photo_url = partial_photo_url['4:3']['400']
+        elsif partial_photo_url['4:3']['200']
+          photo_url = partial_photo_url['4:3']['200']
+        end
+      elsif partial_photo_url['16:9']
+        if partial_photo_url['16:9']['1600']
+          photo_url = partial_photo_url['16:9']['1600']
+        elsif partial_photo_url['16:9']['800']
+          photo_url = partial_photo_url['16:9']['800']
+        elsif partial_photo_url['16:9']['400']
+          photo_url = partial_photo_url['16:9']['400']
+        elsif partial_photo_url['16:9']['200']
+          photo_url = partial_photo_url['16:9']['200']
+        end
+      end
+    else
+      photo_url = 'https://i.guim.co.uk/img/media/26392d05302e02f7bf4eb143bb84c8097d09144b/446_167_3683_2210/master/3683.jpg?width=620&quality=45&auto=format&fit=max&dpr=2&s=6fe0ebd22151102996062fa1287f824c'
+    end
+
+    puts "Determined that photo for #{restaurant['name']} is #{photo_url}"
+    {
+      name: restaurant['venue']['name'],
+      id: restaurant['venue']['id']['resy'],
+      description: restaurant['templates'][default_template_id]['content']['en-us']['about']['body'],
+      latitude: restaurant['venue']['location']['geo']['lat'],
+      longitude: restaurant['venue']['location']['geo']['lon'],
+      submitter_id: 1,
+      photo_url: photo_url
+    }
+  end
+end
+
+unless File.exist?('tomtom-responses.json')
+  mapped_restaurants = non_unique_mapped_restaurants.uniq { |restaurant| restaurant[:latitude] }
+
+  puts 'Generating tomtom responses.json...'
+  restaurant_addresses = mapped_restaurants.map do |restaurant|
+    query_string = "https://api.tomtom.com/search/2/reverseGeocode/#{restaurant[:latitude]},#{restaurant[:longitude]}.json?key=#{Figaro.env.TOMTOM_API_KEY}"
+
+    response = Faraday.get(query_string)
+
+    raise "Error getting address for #{restaurant[:name]}" unless response.status == 200
+
+    puts "Received #{restaurant[:name]} address"
+
+    sleep(1) # sleep because tomtom_api rate limits to one per second
+
+    JSON.parse(response.body)
+  end
+  restaurant_addresses_string = restaurant_addresses.to_json
+
+  File.open('tomtom-responses.json', 'w') { |file| file.write(restaurant_addresses_string) }
+  puts 'Successfully wrote tomtom-responses.json'
+end
+
+unless File.exist?('tomtom-responses.json')
```

Comment:
Very confused by this. You're saying that if the file does not exist, go read it?